app/javascript/packs/application.js

```diff
@@ -47,11 +48,23 @@ class App extends Component {
               path="/appointments/:id/check_in"
               component={CheckIn}
             />
+            <Route
+              path="/users/sign_out"
+            />
           </div>
         </div>
       </Router>
     );
   }
+
+  _signOut() {
+    $.ajax({
+      method: "GET",
+      url: "/users/sign_out",
+    }).done(function(){
+      location.reload();
```

Comment:
TIL `location.reload`! I've always done page reloads in JS through `window.location = location`, which is _ridiculous_ code. 😆 This is so much better. 👍